packages/utils/src/collections.ts

```diff
@@ -91,9 +91,17 @@ export function linkedListChanges(
   const lessers: string[] = []
   const greaters: string[] = []
   for (const it of changeList) {
-    const { lesser, greater } = _linkedListChange(sortedList, it)
+    const { lesser, greater } = _linkedListChange(list, it)
     lessers.push(lesser)
     greaters.push(greater)
   }
   return { lessers, greaters, list }
 }
+
+export function range(from: number, to: number, step: number) {
```

Comment:
❤️

Reply:
We already have this range function, just `import { range } from 'lodash'`